evil-visual-mark-mode.el

```diff
@@ -132,6 +133,18 @@ This function is called when disabling `evil-visual-mark-mode'"
     (lambda (it) (delete-overlay (cdr it))))
   (setq evil-visual-mark-overlay-alist '()))
 
+(defun evil-global-marker-char? (char)
+  (and (>= char ?A) (<= char ?Z)))
```

Comment:
Just wondering, is it a known standard to have capital letters be global? Or is this the scheme that you us in your local setup? Don't mind either way, but, in the case of the latter, can you add some documentation on the README regarding global vs. local markers?

Reply:
Yes, capital letters are for global markers and the manual of vim states: 

> 'a - 'z  lowercase marks, valid within one file
> 'A - 'Z  uppercase marks, also called file marks, valid between files

You could have a try. After setting a marker 'a in both buffer1 and buffer2, you can jump to the marker 'a in each buffer individually. After setting a marker 'A in both buffer1 and buffer2, the marker 'A will be overwritten.

Reply:
By the way, evil is also in the same manner.

Reply:
Ok, cool... this is great work! thank you.